src/HPWH.hh

```diff
@@ -946,6 +949,67 @@ class HPWH
     /// Addition of extra heat handled separately from normal heat sources
     void addExtraHeatAboveNode(double qAdd_kJ, const int nodeNum);
 
+    /// draw-pattern designations for 24-hr test
+    enum class Usage
```

Comment:
Maybe call "UsageCategory"? "Usage" is a little vague. What is the term in the standard?

Reply:
I went with `FirstHourRating` instead.

Reply:
I like it!